stories/components/form/LabeledInput.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Canvas, Story, ArgsTable, Source } from '@storybook/addon-docs';
+
+import LabeledInput from '@/pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput';
+
+export const Template = (args, { argTypes }) => ({
+  components: { LabeledInput },
+  props:      Object.keys(argTypes),
+  template:   '<LabeledInput v-bind="$props" />',
+})
+
+<Meta title="Components/Form/Input" />
+
+# Text input
+Input elements with type ‘text’ allow users to enter any combination of letters, numbers, or symbols. A text box only allows for a single line of input.
+
+
+### Description 
+
+- Always associate a label to every text field for good accessibility. Do not include a colon at the end of the label.
+- Use the appropriate length for the text input. The length of the input text field provides a hint to users as to how much text is expected.
+- An input text field only allows for a single line of input.
+
+
+
+<Canvas>
+  <Story
+    name="LabeledInput"
+    args={{
+      label: 'Label:',
+      value: 'Simon'
+    }}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<Source
+  language='js'
+  code={`
+     import LabeledInput from '@/pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput';
+  `}
+/>
+
+### Props table
+
+<ArgsTable of={LabeledInput} />
+
```

Comment:
Good to see what different status looks like 

`string: info, success, warning, error`

Reply:
Fixed